tensorflow/python/keras/optimizer_v2/adadelta.py

```diff
@@ -45,22 +45,18 @@ class Adadelta(optimizer_v2.OptimizerV2):
   don't have to set an initial learning rate. In this version, initial
   learning rate can be set, as in most other Keras optimizers.
 
-  Args:
-    learning_rate: A `Tensor`, floating point value, or a schedule that is a
-      `tf.keras.optimizers.schedules.LearningRateSchedule`. The learning rate.
-      To match the exact form in the original paper use 1.0.
-    rho: A `Tensor` or a floating point value. The decay rate.
-    epsilon: A `Tensor` or a floating point value.  A constant epsilon used
-             to better conditioning the grad update.
-    name: Optional name prefix for the operations created when applying
-      gradients.  Defaults to `"Adadelta"`.
-    **kwargs: Keyword arguments. Allowed to be one of
-      `"clipnorm"` or `"clipvalue"`.
-      `"clipnorm"` (float) clips gradients by norm; `"clipvalue"` (float) clips
-      gradients by value.
-
-  Reference:
```

Comment:
I think you should revert this, this resolves the merge conflict by going to the previous version, instead of latest version

Reply:
Reverted.Thanks!